www/docs/en/9.x/guide/platforms/electron/index.md

```diff
@@ -248,77 +254,75 @@ The `package` property is an array list of packages to be outputted. If the prop
-| dmg          | -                  | &#10004;           | -                                     |
-| mas          | -                  | &#10004;           | -                                     |
-| mas-dev      | &#10004;           | -                  | -                                     |
-| pkg          | -                  | &#10004;           | -                                     |
-| 7z           | &#10004;           | &#10004;           | &#10004;                              |
-| zip          | &#10004;           | &#10004;           | &#10004;                              |
-| tar.xz       | &#10004;           | &#10004;           | &#10004;                              |
-| tar.lz       | &#10004;           | &#10004;           | &#10004;                              |
-| tar.gz       | &#10004;           | &#10004;           | &#10004;                              |
-| tar.bz2      | &#10004;           | &#10004;           | &#10004;                              |
-| dir          | &#10004;           | &#10004;           | &#10004;                              |
-| nsis         | -                  | -                  | &#10004;                              |
-| nsis-web     | -                  | -                  | &#10004;                              |
-| portable     | -                  | -                  | &#10004;                              |
-| appx         | -                  | -                  | &#10004; <sup>**[1]**</sup>           |
-| msi          | -                  | -                  | &#10004;                              |
-| AppImage     | &#10004;           | -                  | -                                     |
-| snap         | &#10004;           | -                  | -                                     |
-| deb          | &#10004;           | -                  | -                                     |
-| rpm          | &#10004;           | -                  | -                                     |
-| freebsd      | &#10004;           | -                  | -                                     |
-| pacman       | &#10004;           | -                  | -                                     |
-| p5p          | &#10004;           | -                  | -                                     |
-| apk          | &#10004;           | -                  | -                                     |
-
-* **[1]** Only Window 10 can build AppX packages.
-
-#### Setting the package `arch`
+| Package Type |  Linux  |      macOS       |          Windows           |
+| ------------ | :-----: | :--------------: | :------------------------: |
+| default      |    -    | `dmg`<br />`zip` |             -              |
+| dmg          |    -    |     &#9989;      |             -              |
+| mas          |    -    |     &#9989;      |             -              |
+| mas-dev      | &#9989; |        -         |             -              |
```

Comment:
the diff here looks different than on cordova-electron. were there some changes made in cordova-docs only that were not reflected in cordova-electron or is there something funky going on?

Reply:
The content should be exactly identical with the exceptions of:
- The table of contents does not exist here.
- The section "Customizing the Application's Icon" does not exist in the Electron section. All the icon related docs for all platforms are pulled into another file, sadly...

The original changes were made in this PR https://github.com/apache/cordova-electron/pull/61 and then copied to `cordova-docs/dev` and then snapshotted for `9.x`

Reply:
What tripped me up was that on docs the diff is from `:white_check_mark:`, here from another `&#....;`: https://github.com/apache/cordova-electron/pull/61/files#diff-dc89fdcc242c31d756fdf63cc2b6a434L349

Reply:
Ah, yeah it should no longer have `:white_check_mark:` anywhere now. :)

Reply:
Why doesn't have cordova-docs `:white_check_mark:` before this PR, but cordova-electron does? That's what is confusing to me.

Reply:
There was a previous PR in cordova-docs to replace `:white_check_mark:` with the other format because `:white_check_mark:` did not display on the Cordova website. When that change was made to fix the website, it was not changed in `cordova-electron` so it became out of sync. These 2 new PRs now synced them back.

Reply:
Here was the previous PR: https://github.com/apache/cordova-docs/pull/944